src/inject/GPMInject/interface/customUI.js

```diff
@@ -182,13 +182,40 @@ function installBackButton() {
   };
 
   backBtn.addEventListener('click', attemptBack);
+
+  /*
+   OSX has a standard of cmd+left and cmd+right for browser navigation.
+   However cmd does not show up as a modifier key, but rather as a
+   separate keypress, which also eats the keyup event of the key it
+   is modifying.
+   */
+  let isHoldingCmdKey = false;
+
+  window.addEventListener('keydown', (e) => {
+    if (e.which === 91 || e.which === 93) {
+      isHoldingCmdKey = true;
+    }
+
+    if (isHoldingCmdKey && document.activeElement.value === undefined) {
+      if (e.which === 37) {
+        attemptBack();
+      } else if (e.which === 39) {
+        attemptForward();
+      }
+    }
+  });
+
   window.addEventListener('keyup', (e) => {
     if ((e.which === 8 && document.activeElement.value === undefined)
       || (e.which === 37 && e.altKey && document.activeElement.value === undefined)) {
       attemptBack();
     } else if (e.which === 39 && e.altKey && document.activeElement.value === undefined) {
       attemptForward();
     }
+
+    if (e.which === 91 || e.which === 93) {
+      isHoldingCmdKey = false;
```

Comment:
Purely because I enjoy being nitpicky.

Isn't it feasibly possible that a user has 2 "command" keys on their keyboard (Left / Right).  In which case with this logic.

Start = false
Down Left = true
Down Right = true
Up Right = false

Even though you still are holding a command key down we thing they aren't.  Maybe store two vars for left and right or handle the logic a bit differently